shell/platform/android/io/flutter/plugin/platform/PlatformViewWrapper.java

```diff
@@ -308,7 +309,7 @@ public void draw(Canvas canvas) {
         if (Build.VERSION.SDK_INT >= 29) {
           surfaceCanvas.drawColor(Color.TRANSPARENT, BlendMode.CLEAR);
         } else {
-          surfaceCanvas.drawColor(Color.TRANSPARENT);
+          surfaceCanvas.drawColor(Color.TRANSPARENT, PorterDuff.Mode.CLEAR);
```

Comment:
These blocks can be simplified by using drawPaint and setting the color and blend mode on the paint. That's all the Android impl is doing here anyway

Reply:
You then won't have the branch here or need extra tests for different api levels

Reply:
Are you referring to this method? https://developer.android.com/reference/android/graphics/Paint#setBlendMode(android.graphics.BlendMode).

Unfortunately, it also requires API 29.

Although, I agree that we don't need both methods. The `PorterDuf` blend mode isn't deprecated so I think we could just switch to that.

Reply:
Ahh ok, I didn't realize that. This looks good to me though.